recipes/libglvnd/meta.yaml

```
@@ -0,0 +1,152 @@
+        - ninja
+      host:
+        - xorg-libx11
+        - xorg-libxext
+        - xorg-xextproto
+        - xorg-glproto
+    test:
+      commands:
+        - test -f $PREFIX/lib/libEGL.so.1
+        - test -f $PREFIX/lib/libGL.so.1
+        - test -f $PREFIX/lib/libGLESv1_CM.so.1
+        - test -f $PREFIX/lib/libGLESv2.so.2
+        - test -f $PREFIX/lib/libGLX.so.0
+        - test -f $PREFIX/lib/libOpenGL.so.0
+        - test -f $PREFIX/lib/libGLdispatch.so.0
+        - test ! -f $PREFIX/lib/pkgconfig/egl.pc
+        - test ! -f $PREFIX/lib/pkgconfig/gl.pc
+        - test ! -f $PREFIX/lib/pkgconfig/glesv1_cm.pc
+        - test ! -f $PREFIX/lib/pkgconfig/glesv2.pc
+        - test ! -f $PREFIX/lib/pkgconfig/glx.pc
+        - test ! -f $PREFIX/lib/pkgconfig/opengl.pc
+        - test ! -f $PREFIX/lib/pkgconfig/libglvnd.pc
+        - test ! -f $PREFIX/include/EGL/egl.h
+        - test ! -f $PREFIX/include/EGL/eglext.h
+        - test ! -f $PREFIX/include/EGL/eglplatform.h
+        - test ! -f $PREFIX/include/GL/gl.h
+        - test ! -f $PREFIX/include/GL/glcorearb.h
+        - test ! -f $PREFIX/include/GL/glext.h
+        - test ! -f $PREFIX/include/GL/glx.h
+        - test ! -f $PREFIX/include/GL/glxext.h
+        - test ! -f $PREFIX/include/GLES/egl.h
+        - test ! -f $PREFIX/include/GLES/gl.h
+        - test ! -f $PREFIX/include/GLES/glext.h
+        - test ! -f $PREFIX/include/GLES/glplatform.h
+        - test ! -f $PREFIX/include/GLES2/gl2.h
+        - test ! -f $PREFIX/include/GLES2/gl2ext.h
+        - test ! -f $PREFIX/include/GLES2/gl2platform.h
+        - test ! -f $PREFIX/include/GLES3/gl3.h
+        - test ! -f $PREFIX/include/GLES3/gl31.h
+        - test ! -f $PREFIX/include/GLES3/gl32.h
+        - test ! -f $PREFIX/include/GLES3/gl3ext.h
+        - test ! -f $PREFIX/include/GLES3/gl3platform.h
+        - test ! -f $PREFIX/include/KHR/khrplatform.h
+        - test ! -f $PREFIX/include/glvnd/GLdispatchABI.h
+        - test ! -f $PREFIX/include/glvnd/libeglabi.h
+        - test ! -f $PREFIX/include/glvnd/libglxabi.h
+  - name: libgl-devel
+    script: build_libgl_devel.sh
+    build:
+      run_exports:
+        - {{ pin_subpackage('libglvnd', max_pin='x') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+        - meson
+        - pkg-config
+        - ninja
+      host:
+        - xorg-libx11
+        - xorg-libxext
+        - xorg-xextproto
+        - xorg-glproto
+        - libglvnd
+      run:
+        - xorg-xextproto
+        - xorg-glproto
+        - {{ pin_subpackage('libglvnd', exact=True) }}
+    test:
+      commands:
+        - test -f $PREFIX/lib/pkgconfig/egl.pc
+        - test -f $PREFIX/lib/pkgconfig/gl.pc
+        - test -f $PREFIX/lib/pkgconfig/glesv1_cm.pc
+        - test -f $PREFIX/lib/pkgconfig/glesv2.pc
+        - test -f $PREFIX/lib/pkgconfig/glx.pc
+        - test -f $PREFIX/lib/pkgconfig/opengl.pc
+        - test -f $PREFIX/lib/pkgconfig/libglvnd.pc
```

For pkgconfig we could actually go a step further than just testing mere presence of the files and test that the files are functional when loaded/queried (no missing dependencies etc.)
```suggestion
      requires:
        - pkg-config
      commands:
        {% for each_lib in libraries %}
        - pkg-config --print-errors --exact-version "{{ version }}" {{ each_lib | lower }}
        {% endfor %}
```